lucille-core/src/main/java/com/kmwllc/lucille/util/ElasticsearchUtils.java

```diff
@@ -125,4 +129,79 @@ public static boolean getAllowInvalidCert(Config config) {
     }
     return false;
   }
+
+  public static class ElasticJoinData {
```

Comment:
should move to a top-level class
since this is only used by ElasticSearchIndexer it can go in the indexer package itself

ElasticsearchUtils is meant to be a holder of static utility methods relating to elastic, but not a place to nest inner classes